linode/manager
# M3-6174: Remove the `ip_whitelist_enabled` Profile Toggle

## Description 📝

- Removes the deprecated `ip_whitelist_enabled` feature 

## Preview 📷

The `IP Allowlisting (Legacy)` section, which nobody has probably ever seen, will be permanently gone
![Screen Shot 2023-02-16 at 12 26 49 PM](https://user-images.githubusercontent.com/115251059/219441744-5e4fc4f7-a2bb-49c4-bb2d-09da1aeb7966.jpg)

## How to test 🧪

- Before these changes you could use the MSW by adding `ip_whitelist_enabled: true` to your profile. Doing this now will do nothing
- Verify that everything on the `/profile/auth` route works as you'd expect
